src/apps/tools/OswAppDistStats.cpp

```diff
@@ -9,6 +9,8 @@
 #include <osw_hal.h>
 #include <osw_pins.h>
 
+#define WEEK 7
+
```

Comment:
I do not think define-ing the number of week days is really that useful - especially as this define is never reuzsed and just re-defined in each source file. If you really want to label it (and I stretch: I you _really_ want to), maybe declare a `constexpr` int before the `for` call and avoid using the pre-processor for this.